db/system_distributed_keyspace.hh

```diff
@@ -55,8 +55,11 @@ public:
     /* Nodes use this table to communicate new CDC stream generations to other nodes. */
     static constexpr auto CDC_TOPOLOGY_DESCRIPTION = "cdc_generation_descriptions";
 
-    /* This table is used by CDC clients to learn about avaliable CDC streams. */
-    static constexpr auto CDC_DESC = "cdc_streams_descriptions";
+    /* This table is used by CDC clients to learn about available CDC streams. */
+    static constexpr auto CDC_DESC = "cdc_streams_descriptions_v2";
+
```

Comment:
That can be super confusing for someone debugging the code and looking at the wrong version. Not to mention upgrade support.